Code/Tools/FBuild/FBuildCore/BFF/Functions/FunctionLibrary.cpp

```diff
@@ -72,7 +74,8 @@ FunctionLibrary::FunctionLibrary()
 		return false;
 	}
 
-    PathUtils::FixupFolderPath( compilerOutputPath );
+	PathUtils::FixupFolderPath( compilerInputPath );
```

Comment:
(I'll fix this as part of the intergation)

This actually needs to be like this:
```C++
	if ( !compilerInputPath.IsEmpty() )
	{
		AStackString<> tmp;
		NodeGraph::CleanPath( compilerInputPath, tmp );
		compilerInputPath = tmp;
	}
```
Without the IsEmpty() check, compilerInputPath would not be empty by default (FixupFolderPath would always add a trailing slash).  Also making this a CleanPath here removes the need for cleaning the path at build time.